docs/source/guides/contribution_guide.md

```diff
@@ -0,0 +1,68 @@
+# Contributing
+
+![NuCypher Unicorn](https://cdn-images-1.medium.com/max/800/1*J31AEMsTP6o_E5QOohn0Hw.png)
+
+
+## Running the Tests
+
+``` note:: A development installation including the solidity comppiler is required to run the tests
```

Comment:
```suggestion
``` note:: A development installation including the solidity compiler is required to run the tests
```